scheduler-driver/src/main/java/gov/nasa/jpl/aerie/scheduler/simulation/ResumableSimulationDriver.java

```diff
@@ -196,9 +218,12 @@ public Duration getCurrentSimulationEndTime(){
   public SimulationResults getSimulationResultsUpTo(Instant startTimestamp, Duration endTime){
     //if previous results cover a bigger period, we return do not regenerate
     if(endTime.longerThan(curTime)){
+      logger.info("Has to simulate from " + curTime + " to " + endTime + " to get simulation results");
```

Comment:
Minor, can ignore: conciseness 
```suggestion
      logger.info("Simulating from " + curTime + " to " + endTime);
```

Reply:
Updated partially this message.